src/components/map/index.js

```diff
@@ -1,21 +1,56 @@
-// eslint-disable-next-line no-unused-vars
 import { MapContainer, TileLayer, Marker, Popup } from 'react-leaflet';
+import marker from './icon';
+import { useEffect, useState } from 'react';
+
 import 'leaflet/dist/leaflet.css';
 import './styles.scss';
 
 const MapView = () => {
+  const [position, setPosition] = useState({
+    where: [],
+    error: null,
+  });
+  const geoOptions = {
+    enableHighAccuracy: true,
+    timeout: 5000,
+    maximumAge: 0,
+  };
+  const geolocationEnabled = () => {
+    setPosition({ error: null });
+    return navigator.geolocation.getCurrentPosition(
+      position => {
+        geoSuccess(position);
+      },
+      () => {},
+      geoOptions
+    );
+  };
+
+  const geoSuccess = position => {
+    setPosition({
+      ready: true,
```

Comment:
I've seen differences in what the `position` object will be like during the component lifecycle.

In its initial state the shape of position is:

```js
{
 where: [],
 error: null
}
```
whereas here for example you are adding a `ready` prop to it. And in the `geolocationEnabled` you are setting an object with just the `error` prop in it. What do you think about making the necessary changes to keep consistency in it?